packages/app/src/components/Bookmarks/BookmarkFolderMenuItem.tsx

```diff
@@ -198,6 +199,15 @@ const BookmarkFolderMenuItem = (props: Props): JSX.Element => {
         {renderBookmarkSubMenuItem()}
 
       </UncontrolledDropdown >
+      <UncontrolledTooltip
+        modifiers={{ preventOverflow: { boundariesElement: 'window' } }}
+        autohide={false}
+        placement="top"
+        target={`bookmark-delete-button-${item._id}`}
+        fade={false}
+      >
+        {t('bookmark_folder.delete')}
+      </UncontrolledTooltip>
```

Comment:
@mudana-grune 
A `DeleteBookmarkFolderModal` should appear when the delete button is pressed.